.build

```diff
@@ -1,4 +1,5 @@
 CC = gcc
```

Comment:
why a .file however? That stuff is a cancer. Why not a file named ENV or DEFAULTS or something reasonably official?

overall however I am not convinced this is a good idea. It's not that hard to
export ARCH=amd64
and then not worry. But it's your tool, up to you, that's why I approved it.
Also, your .build hardwires amd64 and in general hardwire ARCH in any file is bad policy, especially one checked in to git.